presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestTransformCorrelatedSingleRowSubqueryToProject.java

```diff
@@ -50,7 +54,10 @@ public void testRewrite()
                                 ImmutableList.of(p.symbol("l_nationkey")),
                                 p.tableScan(new TableHandle(
                                                 new ConnectorId("local"),
-                                                new TpchTableHandle("nation", TINY_SCALE_FACTOR)), ImmutableList.of(p.symbol("l_nationkey")),
+                                                new TpchTableHandle("nation", TINY_SCALE_FACTOR),
+                                                TestingTransactionHandle.create(),
```

Comment:
Shall we use `TpchTransactionHandle.INSTANCE` or `TestingTransactionHandle.create()` here?  I would left this question for @arhimondr since he wrote `TestingTransactionHandle` ;)